### ISSUE TITLE
Ui doesn't connect to the auth emulator

### ISSUE THREAD

**Issue by Piepongwong** [firebase/firebaseui-web-react#118]
I'm running a create react app with the firestore, functions, database and authentication emulators enabled. 

When I try to signup a user, it connects to the live authentication service. The other emulators do work and I also do get the `Running in emulator mode. Do not use with production credentials.` message. If I try [non interactive testing](https://firebase.google.com/docs/emulator-suite/connect_auth#non-interactive_testing_3) it does connect to the emulator. 

This seems to be a bug or undescribed behaviour although I'm relatively new to Firebase.

**Comment by josueetcom** [comment]
I'm also experiencing this issue. From what the [auth emulator docs](https://firebase.google.com/docs/emulator-suite/connect_auth#instrumentauth) suggest, it seems like it _should_ be as simple as:
```js
// initialize app
firebase.auth().useEmulator('http://localhost:9099/');
``` 
and the API key + auth domain shouldn't be required?

**Comment by josueetcom** [comment]
It seems like the issue might be in the underlying Firebase Web UI library as there's a similar issue:
https://github.com/firebase/firebaseui-web/issues/778

**Comment by interactive-web-dev** [comment]
This package's only dependency [firebaseui](https://www.npmjs.com/package/firebaseui) has fixed the underlying issue [firebase/firebaseui-web#778](https://github.com/firebase/firebaseui-web#778) as part of the latest release [v4.7.2+](https://github.com/firebase/firebaseui-web/releases/tag/v4.7.2). Does this package require any development to enable local auth emulation? Or is it just a case of updating the dependency?

**Comment by interactive-web-dev** [comment]
I made a local patch, and it seems to be working! The local emulator is a real game changer :D.
For anyone just looking to get the feature up and running quickly:

Update and save the **firebaseui** dependency inside your local installation of the **react-firebaseui** package.
`cd <package_path> && npx i firebaseui@latest --save`

Optionally, you can keep a record of the temporary patch.
`npx patch-package react-firebaseui`.

**Comment by alexmcmanus** [comment]
If you are using Yarn, add this to `package.json`:
```json
  "resolutions": {
    "react-firebaseui/firebaseui": "^4.7.3"
  },
```
Then run `yarn install` again.

**Comment by AskYous** [comment]
Any updates to this issue?

**Comment by jhuleatt** [comment]
This has been fixed in `react-firebaseui@5.0.1`
